src/ABC_Datalog/main.pl

```diff
@@ -0,0 +1,447 @@
+/*
+Date: 07 Jan 2019
+Macintosh HD⁩/⁨Users⁩/lixue⁩/GoogleDrive⁩/01PHD⁩/01program⁩/eclipse-workspace⁩/ABC_Clean⁩/src⁩
+*/
```

Comment:
Can we not include this comment? Seems highly specific to your working directory